### PR TITLE
BUG: Fixed PagedJobsTable Storybook

### PR DESCRIPTION
**Pull request by iain801**
## Overview

In this PR, we fix the PagedJobsTable component to accommodate the storybook props.

## Screenshots (Optional)

image

## Validation (Optional)

1. Download this branch.
2. Run the storybook.
3. Go to this page.

## Tests

- [ ] Backend Unit tests (`mvn test`) pass
- [ ] Backend Test coverage (`mvn test jacoco:report`) 100%
- [ ] Backend Mutation tests (`mvn test pitest:mutationCoverage`) 100% 
- [ ] Frontend Unit tests (`npm test`) pass
- [ ] Frontend Test coverage (`npm run coverage`) 100%
- [ ] Frontend Mutation tests (`npx stryker run`) 100% 
- [ ] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes None

### CODE REVIEW

**Review by dscpsyl — DISMISSED** [review]
Looks to be simple fix

File: frontend/src/main/components/Jobs/PagedJobsTable.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import OurTable, { PlaintextColumn, DateColumn } from "main/components/OurTable"
 import { Button } from "react-bootstrap";
 import { useBackend } from "main/utils/useBackend";
 
-export default function PagedJobsTable() {
+const PagedJobsTable = ({jobs: jobsProp}) => {
```
Comment (chriscerie) [inline frontend/src/main/components/Jobs/PagedJobsTable.js]: Renaming a prop to append “Props” doesn’t tell you anything. If it’s for stories, give it a meaningful name that says so. Or use msw.

File: frontend/src/main/components/Jobs/PagedJobsTable.js [hunk 2]
```diff
@@ -30,6 +30,8 @@ export default function PagedJobsTable() {
     );
     // Stryker restore  all
 
+    page.content = jobsProp || page.content;
```
Comment (chriscerie) [inline frontend/src/main/components/Jobs/PagedJobsTable.js]: Don’t mutate this.

**Review by chriscerie — CHANGES_REQUESTED** [review]

File: frontend/src/main/components/Jobs/PagedJobsTable.js (hunk 1 above)
Comment (pconrad) [reply frontend/src/main/components/Jobs/PagedJobsTable.js]: @chriscerie has a good point.   

I'm not sure if the only point of the props is for storybook, then we should use msw instead.  That will be a better test of how the component is being used in practice.

File: frontend/src/main/components/Jobs/PagedJobsTable.js (hunk 2 above)
Comment (pconrad) [reply frontend/src/main/components/Jobs/PagedJobsTable.js]: Yeah, it's weird to mutate the variable that's being returned from a React Query call, especially if the purpose is for storybook.

We have a pattern for that, and that pattern is using msw (see the examples in the team03 assignment.)

File: frontend/src/stories/components/Jobs/PagedJobsTable.stories.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import PagedJobsTable from "main/components/Jobs/PagedJobsTable";
 import jobsFixtures from "fixtures/jobsFixtures";
 
 export default {
-    title: 'components/Leaderboard/PagedJobsTable',
+    title: 'components/Jobs/PagedJobsTable',
```
Comment (pconrad) [inline frontend/src/stories/components/Jobs/PagedJobsTable.stories.js:8]: This is a good fix though.

**Review by pconrad — DISMISSED** [review]
Can we change this to use the msw pattern instead of this, which feels like a hack and may confuse future devs?

**Review by dscpsyl — APPROVED** [review]
Changes look much better

**Review by GDW1 — APPROVED** [review]

**Pull request merged**